### ISSUE TITLE
Electroblob Wizardry Bookshelf Replaced by Lootr Chest

### ISSUE THREAD

**Issue by reddvilzz** [noobanidus/Lootr#119]
The bookshelf need to be added to the blacklist or you can't open them at all.

Might be compatibility issue?

I'm on 1.12.2

**Comment by embeddedt** [comment]
Unfortunately I'm not sure I can do anything to prevent this automatically. Evidently the bookshelves are using loot tables and being picked up as a loot-based block. 1.12 doesn't have tags so it's hard to know what is a chest and what isn't.

I could however add them to the blacklist by default; what is their registry name?

**Comment by reddvilzz** [comment]
I added this

> ebwizardry:oak_bookshelf
        ebwizardry:spruce_bookshelf
        ebwizardry:birch_bookshelf
        ebwizardry:jungle_bookshelf
        ebwizardry:acacia_bookshelf
        ebwizardry:dark_oak_bookshelf
        tfspellpack:twilight_oak_bookshelf
        tfspellpack:canopy_bookshelf
        tfspellpack:mangrove_bookshelf
        tfspellpack:darkwood_bookshelf
        tfspellpack:timewood_bookshelf
        tfspellpack:transwood_bookshelf
        tfspellpack:minewood_bookshelf
        tfspellpack:sortingwood_bookshelf

Also I added this 
>
    S:LOOT_MODID_BLACKLIST <
		iceandfire
		ebwizardry
		w_tweaks
     >

Since Ice and Fire, Electroblob's Wizardry, and Wizardry Tales behave the same when their chests turned into Lootr chests they cannot be opened, and If I break it will just return an empty chest. So that's the correct ways to blacklist a mod?

**Comment by embeddedt** [comment]
> Since Ice and Fire, Electroblob's Wizardry, and Wizardry Tales behave the same when their chests turned into Lootr chests they cannot be opened

Okay, *that* is a bug. The chest should still be possible to open.

**Comment by reddvilzz** [comment]
Do I need to provide any logs to help?

**Comment by embeddedt** [comment]
I don't need any logs, but could you quickly explain how I can find a naturally generated bookshelf/block from each of these mods? I've never played with any of them so that would speed up reproduction greatly.

**Comment by reddvilzz** [comment]
Basically it just spawned in randomly, maybe it's easier to do it either in flat world while increase the spawn chance in the config files? Personally I never done that but when I fly around in creative they are quite easy to find.

**Comment by embeddedt** [comment]
Is this the type of chest that wouldn't open? It seemed to work for me. Perhaps a log would be helpful after all.

![2022-07-02_16 09 24](https://user-images.githubusercontent.com/42941056/177014895-dac3b23b-98f6-4280-9986-fb74a7ae1cc5.png)

**Comment by reddvilzz** [comment]
okay I tried reproducing it on flat world myself and it seems you are right, the chests that I mentioned before can be opened. Idk if that's because I use Chunk Pregenerator mod on my world before since I don't use Chunk Pregenerator on the flat world.

I tested it out with a new world without pregening and all the chests can be opened. I guess that's solved.

Also I found out chests from Recurrent Complex Structures also not replaced by Lootr chests, is there anyway I can force it to change? Sorry if it's going out of topic from the title mentioned, do I need to post a new issue for this?
